.ci/install_linux_cmake.sh

```diff
@@ -39,7 +36,7 @@ if [ "$BUILD_AIKIDOPY" = "ON" ]; then
   $SUDO apt-get -y install python3-pip -y
   $SUDO pip3 install pytest -U
 
-  if [ $(lsb_release -sc) = "trusty" ] || [ $(lsb_release -sc) = "xenial" ] || [ $(lsb_release -sc) = "bionic" ]; then
+  if [ $(lsb_release -sc) = "bionic" ]; then
```

Comment:
Clearly there is some support for earlier versions of Ubuntu. So maybe we should just document which versions of Ubuntu are supported? (Maybe you already did and I haven't seen that file yet)

Reply:
Recall that these are Continuous-Integration workflows. They are only executed on the OS's that we tell them to execute on, and right now, we tell them to execute on bionic and focal. We could leave these statements in, but they will never be used again, since these versions are end-of-life (Github does maintain them in github-actions).